CHANGELOG.md

```diff
@@ -0,0 +1,125 @@
+### 1.8.0 - 2016-06-21
+
+- Added `--version` and `-v` options to the CLI now has that show the version of
+  Neat you have installed
+- Neat no longer requires Bourbon
+
+### 1.7.4 - 2016-03-04
+
+- Added an explicit sass requirement
+
+### 1.7.3 2016-02-29
+
+- Neat's npm package now includes the proper Sass paths
+- Added eyeglass support
+- The default `$max-width` is now `1200px`
+
+### 1.7.2 - 2015-03-13
+
+- Removed prefixes for `box-sizing`
+- Added Bourbon's updated `modular-scale()` function for grid settings
+
+### 1.7.1 - 2015-01-02
+
+- Added custom `not()` function for Libsass 3.0 compatibility
+- Fixed bug related to the `omega()`'s query argument
+
+### 1.7.0 2014-10-17
+
+- `direction-context()` mixin changes the direction inside an entire block.
+- Removed `reset-direction()`
+- `display-context()` mixin changes the display property inside an entire block.
+- Removed `reset-display()`
+- `outer-container()` now takes a max-width argument
+- Full compatibility with Sass 3.4+ and Libsass 3.0+
+
+### [1.5.1] 2014-04-06
+
+- Restore compatibility with Sass 3.2.x
+
+### [1.6.0] 2014-03-28
+
+- Added multiple media queries support to the `media()` mixin, including queries
+  such as orientation
+- Updated to Sass 3.3 and silence `!global` variable warnings
+- Use absolute percentage widths for table displays
+- Passing table no longer adds padding to elements
+- Omega no longer supports table display
+
+### [1.6.0.pre2] - 2014-03-21
+
+- Use Sass 3.3 and silence global variable warnings
+
+### [1.6.0.pre] 2014-02-10
+
+- Added multiple media queries support to the `media()` mixin, including queries
+  such as orientation
+- Use absolute percentage widths for table displays
+- Passing table no longer adds padding to elements
+- Omega no longer supports table display
+
+### [1.5.0] 2013-22-15
+
+- Improved browser support
+- Renamed `span-columns()` collapse argument to block-collapse
+- Added the ability for `span-columns()` to omit display property from the output
+- Made `shift()` take parent column argument (useful when other `span-columns()`
+  calls override the nesting context)
+
+### [1.4.0] - 2013-08-28
+
+- Added collapse argument to `span-columns()` in order to remove gutter
+- Fixed visual grid overrides by reordering breakpoints
+- Unified padding across table cells
+- Specified comma as separator for gradient-stops append for libsass
+
+### [1.3.0] - 2013-05-03
+
+- Added global RTL layout support
+- Added row-specific RTL layout support
+- Added auto-clearing for elements using `omega(nth-child)` (works only with
+  simple `nth-child()` arguments due to limited string operations in Sass)
+- `reset-display`, `reset-layout-direction`, and `reset-all` for all your reset needs!
+- The visual grid no longer blocks user interactions
+- `omega()` will no longer take `$direction` in favor of `row($direction)` or
+  `$default-layout-direction`
+
+### [1.2.1] - 2013-04-04
```

Comment:
So I was trying to make links for all the versions but for some reason it does not create a diff. 

See: https://github.com/thoughtbot/neat/compare/1.7.4...v1.8.0

I guess I'll just have to remove them :(

[1.8.0]: https://github.com/thoughtbot/neat/compare/1.7.4...v1.8.0
[1.7.4]: https://github.com/thoughtbot/neat/compare/1.7.3...v1.7.4
[1.7.3]: https://github.com/thoughtbot/neat/compare/1.7.2...v1.7.3
[1.7.2]: https://github.com/thoughtbot/neat/compare/1.7.1...v1.7.2
[1.7.1]: https://github.com/thoughtbot/neat/compare/1.7.0...v1.7.1
[1.7.0]: https://github.com/thoughtbot/neat/compare/1.5.1...v1.7.0
[1.5.1]: https://github.com/thoughtbot/neat/compare/1.6.0...v1.5.1
[1.6.0]: https://github.com/thoughtbot/neat/compare/1.6.0.pre2...v1.6.0
[1.6.0.pre2]: https://github.com/thoughtbot/neat/compare/1.6.0.pre...v1.6.0.pre2
[1.6.0.pre]: https://github.com/thoughtbot/neat/compare/1.5.0...v1.6.0.pre
[1.5.0]: https://github.com/thoughtbot/neat/compare/1.4.0...v1.5.0
[1.4.0]: https://github.com/thoughtbot/neat/compare/1.3.0...v1.4.0
[1.3.0]: https://github.com/thoughtbot/neat/compare/1.2.1...v1.3.0
[1.2.1]: https://github.com/thoughtbot/neat/compare/1.2.0...v1.2.1
[1.2.0]: https://github.com/thoughtbot/neat/compare/1.1.0...v1.2.0
[1.1.0]: https://github.com/thoughtbot/neat/compare/1.0.2...v1.1.0
[1.0.2]: https://github.com/thoughtbot/neat/compare/1.0.1...v1.0.2
[1.0.1]: https://github.com/thoughtbot/neat/compare/1.0.0...v1.0.1
[1.0.0]: https://github.com/thoughtbot/neat/compare/0.4.2...v1.0.0